### PR TITLE
Provide an example of meta in dbt_project

### PR DESCRIPTION
**Pull request by jsnb-devoted**
## Description & motivation

It took me a while to figure out why my `meta` tag wasn't being captured in the dbt_project file. I needed it nested under the `config` block. I think it would have helped if there was an example of that.

(I deleted all the checklists as it didn't seem like they were needed for this PR)

### CODE REVIEW

**Comment by runleonarun** [comment]
Thank you for contributing to this site! 

We're closing this PR because it looks like we've added `owner` to examples on this page.

**Comment by jsnb-devoted** [comment]
@runleonarun -- my example was doing it in the `dbt_project.yml` file. The syntax is different. It requires nesting it under the `config` options. If you wanted to add an owner to a folder of models you would need to know that the `config` is required.

**Comment by jsnb-devoted** [comment]
dbt_project.yml:
```
models:
  <project>:
    <folder containing models>:
      +config:
        meta:
          owner: <owner name>
```
this is different than how it is done in the example on the site. There is no `config` block ([docs](https://docs.getdbt.com/reference/resource-configs/meta#designate-a-model-owner))

**Comment by runleonarun** [comment]
Thanks for letting me know @jsnb-devoted! I re-opened it and will review and merge it.

File: website/docs/reference/resource-configs/meta.md [hunk 1]
```diff
@@ -278,4 +278,19 @@ select 1 as id
 ```
 
 </File>
+  
+### Assign owner in the dbt_project.yml
```
Comment (runleonarun) [inline website/docs/reference/resource-configs/meta.md]:
```suggestion
### Assign owner in the dbt_project.yml as a config property
```

**Review by runleonarun — APPROVED** [review]
Looks great! Thank you!

**Pull request merged**